### PR TITLE
android-grpc

### PR DESCRIPTION
**Pull request by iZIVer**
* added android-grpc
* deleted old android project

### CODE REVIEW

**Comment by mplooster** [comment]
Most of this was auto-generated, right?

File: grpc/android-cubic/sdk/build.gradle [hunk 1]
```diff
@@ -0,0 +1,69 @@
+apply from: 'preBuilderProtoFiles.gradle'
+
+apply plugin: 'com.android.library'
+apply plugin: 'com.google.protobuf'
+apply plugin: 'com.github.dcendents.android-maven'
+
+group='com.github.tumarsal'
```
Comment (kapitoshka438) [inline grpc/android-cubic/sdk/build.gradle:7]: Should the group name contain `tumarsal`?

File: grpc/android-cubic/sdk/build.gradle [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+android {
+    preBuild.dependsOn initProtoFiles, downloadGoogleApi, downloadGoogleProto
+
+    compileSdkVersion 29
+
+
```
Comment (kapitoshka438) [inline grpc/android-cubic/sdk/build.gradle:13]: Extra empty line

File: grpc/android-cubic/sdk/preBuilderProtoFiles.gradle [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+task downloadGoogleProto(type: Download) {
+    src([
+            'https://raw.githubusercontent.com/protocolbuffers/protobuf/master/src/google/protobuf/descriptor.proto',
+            'https://raw.githubusercontent.com/protocolbuffers/protobuf/master/src/google/protobuf/duration.proto',
+            'https://raw.githubusercontent.com/protocolbuffers/protobuf/master/src/google/protobuf/empty.proto'
+    ])
+    dest 'src/main/proto/google/protobuf'
+    onlyIfModified true
+}
+
```
Comment (kapitoshka438) [inline grpc/android-cubic/sdk/preBuilderProtoFiles.gradle:18]: Extra empty lines

File: grpc/android-cubic/sdk/preBuilderProtoFiles.gradle [hunk 4]
```diff
@@ -0,0 +1,26 @@
+task downloadGoogleApi(type: Download) {
+    src([
+            'https://raw.githubusercontent.com/googleapis/googleapis/master/google/api/annotations.proto',
+            'https://raw.githubusercontent.com/googleapis/googleapis/master/google/api/http.proto'
+    ])
+    dest 'src/main/proto/google/api'
+    onlyIfModified true
+}
+task downloadGoogleProto(type: Download) {
```
Comment (kapitoshka438) [inline grpc/android-cubic/sdk/preBuilderProtoFiles.gradle:9]: Insert empty line

File: grpc/android-cubic/build.gradle [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+        google()
+        jcenter()
+        
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.6.3'
+        classpath 'com.github.dcendents:android-maven-gradle-plugin:2.1' // Add this line
+        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.10'
+    }
+}
```
Comment (kapitoshka438) [inline grpc/android-cubic/build.gradle:15]: I think there should be empty line above. Or there shouldn't be empty line on the 4-th row.

File: grpc/android-cubic/gradle/wrapper/gradle-wrapper.properties [hunk 6]
```diff
@@ -0,0 +1,6 @@
+#Mon May 04 10:54:18 MSK 2020
```
Comment (kapitoshka438) [inline grpc/android-cubic/gradle/wrapper/gradle-wrapper.properties:1]: Do we need this comment?

File: grpc/android-cubic/sdk/src/main/java/cobaltspeech/cubic/CubicGrpc.java [hunk 7]
```diff
@@ -0,0 +1,518 @@
...
+        }
+      }
+    }
+    return getVersionMethod;
+  }
+
+  private static volatile io.grpc.MethodDescriptor<cobaltspeech.cubic.CubicOuterClass.ListModelsRequest,
+      cobaltspeech.cubic.CubicOuterClass.ListModelsResponse> getListModelsMethod;
+
+  @io.grpc.stub.annotations.RpcMethod(
```
Comment (kapitoshka438) [inline grpc/android-cubic/sdk/src/main/java/cobaltspeech/cubic/CubicGrpc.java:66]: It looks like there are some indentation issues with the method below

File: grpc/android-cubic/sdk/src/main/java/cobaltspeech/cubic/CubicGrpc.java [hunk 8]
```diff
@@ -0,0 +1,518 @@
...
+
+  private CubicGrpc() {}
+
+  public static final String SERVICE_NAME = "cobaltspeech.cubic.Cubic";
+
+  // Static method descriptors that strictly reflect the proto.
+  private static volatile io.grpc.MethodDescriptor<com.google.protobuf.Empty,
+      cobaltspeech.cubic.CubicOuterClass.VersionResponse> getVersionMethod;
+
+  @io.grpc.stub.annotations.RpcMethod(
```
Comment (kapitoshka438) [inline grpc/android-cubic/sdk/src/main/java/cobaltspeech/cubic/CubicGrpc.java:36]: It looks like there are some indentation issues with the method below

File: grpc/android-cubic/sdk/src/main/java/cobaltspeech/cubic/CubicGrpc.java [hunk 9]
```diff
@@ -0,0 +1,518 @@
+package cobaltspeech.cubic;
```
Comment (kapitoshka438) [inline grpc/android-cubic/sdk/src/main/java/cobaltspeech/cubic/CubicGrpc.java:1]: I think here and in other sources code files should be a copyrights header.

File: grpc/android-cubic/settings.gradle [hunk 10]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (kapitoshka438) [inline grpc/android-cubic/settings.gradle:1]: Extra empty lines?

File: grpc/android-cubic/sdk/src/main/java/cobaltspeech/cubic/CubicGrpc.java (hunk 7 above)
Comment (tumarsal) [reply grpc/android-cubic/sdk/src/main/java/cobaltspeech/cubic/CubicGrpc.java:66]: It is generated code

File: grpc/android-cubic/sdk/src/main/java/cobaltspeech/cubic/CubicGrpc.java (hunk 8 above)
Comment (tumarsal) [reply grpc/android-cubic/sdk/src/main/java/cobaltspeech/cubic/CubicGrpc.java:36]: It is generated code

**Pull request merged**